src/go/plugin/go.d/config/go.d/snmp.profiles/default/_cisco-asa.yaml

```diff
@@ -12,7 +12,8 @@ metrics:
       - OID: 1.3.6.1.4.1.9.9.147.1.2.2.2.1.5
         name: cfwConnectionStatValue
         description: Current status of the resource statistic
-        unit: "{statistic}"
+        unit: "1"
```

Comment:
What is "1"? What does it mean?

Reply:
Welcome to UCUM units!

Check [examples for non units](https://ucum.org/ucum)

Reply:
Ok 😑 

> Current status of the resource statistic

I think this should be `status`.

Reply:
Yes, `{status}` is fine. Basically this is short for `1{status}`, meaning a dimensionless unit, annotated with "status".